shared/wallets/transaction/index.js

```diff
@@ -0,0 +1,180 @@
+// @flow
+import * as React from 'react'
+import {Avatar, Box2, Text} from '../../common-adapters'
+import {globalColors, globalMargins, platformStyles, styleSheetCreate} from '../../styles'
+// TODO: Format relative dates.
+import {formatTimeForPopup} from '../../util/timestamp'
+
+type Role = 'sender' | 'receiver'
+type CounterpartyType = 'keybaseUser' | 'stellarPublicKey' | 'wallet'
+
+type IconProps = {|
+  large: boolean,
+  counterparty: string,
+  counterpartyType: CounterpartyType,
+|}
+
+const Icon = (props: IconProps) => {
+  switch (props.counterpartyType) {
+    case 'keybaseUser':
+      return <Avatar username={props.counterparty} size={props.large ? 48 : 32} />
+    case 'stellarPublicKey':
+      // TODO: Return anonymous user icon.
```

Comment:
I don't see the icons in `icon.constants.js` yet

Reply:
if its not there you can export it. i belive its just a png

Reply:
Ah, figured it out. Done!